Backend/lib/beckn-spec/src/Domain/Types/FareProductType.hs

```diff
@@ -36,9 +36,10 @@ getFareProductType :: TripCategory -> FareProductType
 getFareProductType tripCategory =
   case tripCategory of
     OneWay OneWayRideOtp -> ONE_WAY_SPECIAL_ZONE
-    Rental OnDemandStaticOffer -> RENTAL
-    InterCity OneWayOnDemandStaticOffer Nothing -> INTER_CITY
-    Ambulance OneWayOnDemandDynamicOffer -> AMBULANCE
+    CrossCity OneWayRideOtp _ -> ONE_WAY_SPECIAL_ZONE
+    Rental _ -> RENTAL
+    InterCity _ _ -> INTER_CITY
+    Ambulance _ -> AMBULANCE
     _ -> DRIVER_OFFER
```

Comment:
This also Specific mapping is not required based, top level mapping of trip category to fareProductType should be fine for `Rental, InterCity and Ambulance`.